tests/integration/test_backups.py

```diff
@@ -430,7 +430,7 @@ async def test_invalid_config_and_recovery_after_fixing_it(
 
 
 @pytest.mark.group(1)
-async def test_delete_pod(ops_test: OpsTest):
+async def test_delete_pod(ops_test: OpsTest, github_secrets) -> None:
```

Comment:
The test depend on the preceding tests, so it should also be skipped if there are no secrets.